firelens/README.md

```diff
@@ -120,7 +120,7 @@ Following is an example task definition with an nginx web server accessible on p
    "containerDefinitions":[
       {
          "essential":true,
-         "image":"splunk/fluentd-hec:1.2.0",
+         "image":"splunk/fluentd-hec:1.2.1",
```

Comment:
in PLUGIN_VERSIONS.sh `FLUENTD_HEC_VERSION=1.3.1` but here its  `1.2.1`?

Reply:
Thanks!